### PR TITLE
Added Connect2All On-Premises

### PR DESCRIPTION
**Pull request by Joerio**
Added new connector 'Connect2All On-Premises'.
Updated readme of existing connector 'Connect2All'.

### CODE REVIEW

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,4024 @@
...
+            "$ref": "#/parameters/Company"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "properties": {
+                "CallbackUrl": {
+                  "description": "CallbackUrl",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json:3178]: @Joerio strings in camel case are considered non-translatable. Please make sure that descriptions look like valid sentences. More information about quality strings can be found in [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,4024 @@
+{
+  "basePath": "/",
+  "consumes": [],
+  "definitions": {
+    "CodeunitId": {
+      "description": "The codeunit you wish to execute",
+      "format": "int32",
+      "title": "",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json:8]: @Joerio empty properties are not allowed. Please look through empty titles and add value to them.

File: certified-connectors/Connect2All On-Premises/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          },
+          "description": "The Username for this api",
+          "displayName": "Username",
+          "tooltip": "Provide the Username"
+        }
+      }
+    },
+    "iconBrandColor": "#FA9901",
+    "publisher": "GAC Business Solutions",
+    "stackOwner":  "GAC Business Solutions"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Connect2All On-Premises/apiProperties.json:34]: @Joerio please remove an empty whitespace here before the value.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @Joerio! Thank you for making a submission! Please check comments after the review, thank you!

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json (hunk 2 above)
Comment (Joerio) [reply certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json]: Added titles to all empty title properties.

File: certified-connectors/Connect2All On-Premises/apiProperties.json (hunk 3 above)
Comment (Joerio) [reply certified-connectors/Connect2All On-Premises/apiProperties.json]: Removed whitespace.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @Joerio, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit id, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**